lib/agent0/examples/interactive_remote_hyperdrive_example.py

```diff
@@ -0,0 +1,76 @@
+"""Example script for using interactive hyperdrive to connect to a remote chain.
+Note that this script only works when there exists an anvil chain running in docker.
```

Comment:
Is this a script requirement or is this an interactive remote hyperdrive requirement? Either way, why?